### ISSUE TITLE
The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"

### ISSUE THREAD

**Issue by huineng** [carbon-design-system/carbon-charts#1724]
### Name

Guy Huinen

### Are you an IBM employee?

- [X] Yes

### Environment

Browser

### What happened? What did you expect to happen instead?

on each of the 3 toolbar controls i get an accessibility issue

![image](https://github.com/carbon-design-system/carbon-charts/assets/3482445/7fb09620-777d-4145-b41d-9dc507183b1e)

ps you can see the same results if you do a scan using the IBM equal access accessibility checker and run it on any chart with a toolbar on https://charts.carbondesignsystem.com/

thanks

### What WCAG 2.1 checkpoint does the issue violate?

Rule ID: aria_attribute_valid Reason ID: Fail_invalid_implicit_role_attr

### Version

1.13.18

### What priority level would this issue be in your opinion?

P0
